cfme/tests/configure/test_access_control.py

```diff
@@ -94,8 +90,11 @@ def _check_item_visibility(item, user_restricted):
 
 # User test cases
 @pytest.mark.tier(2)
-def test_user_crud():
-    user = new_user()
+def test_user_crud(group_collection):
+    group_name = 'EvmGroup-user'
+    group = group_collection.instantiate(description=group_name)
```

Comment:
**REQUIRED**
These two lines can be a fixture as well.

Reply:
I'm choosing to leave this as it is. I am working on a follow up PR that will be using pytest parameters to run individual tests against multiple roles/groups/users which will address your suggestion.